test/DataCollectors/TraceDataCollector.UnitTests/DynamicCoverageDataCollectorImplTests.cs

```diff
@@ -83,6 +87,48 @@ public void InitializeShouldCreateDefaultCodeCoverageSettingsIfConfigElementIsNu
             this.CompareWithDefaultConfig();
         }
 
+        [TestMethod]
+        public void InitializeShouldGenerateCodeCoverageDepsJsonFile()
+        {
+            this.directoryHelperMock.Setup(d => d.CreateDirectory(It.IsAny<string>()))
+                .Callback<string>((path) => Directory.CreateDirectory(path));
+
+            this.fileHelperMock.Setup(f => f.WriteAllText(It.IsAny<string>(), It.IsAny<string>()))
+                .Callback<string, string>((path, content) => { File.WriteAllText(path, content); });
+
+            this.profilersLocationProviderMock.Setup(lp => lp.GetCodeCoverageShimPath()).Returns(@"C:\aaa\bbb\Microsoft.VisualStudio.CodeCoverage.Shim.dll");
+
+            this.collectorImpl.Initialize(null, this.dataCollectionSinkMock.Object, this.dataCollectionLoggerMock.Object);
+
+            var obj = JObject.Parse(File.ReadAllText(this.collectorImpl.CodeCoverageDepsJsonFilePath));
+
+            var expected = @"
```

Comment:
The same as above.

Reply:
Since it's a json file, an new line shouldn't affect anything other than formatting.

Reply:
also this stuff is working only on Windows now